qiskit_experiments/framework/backend_timing.py

```diff
@@ -180,21 +180,46 @@ def circuits(self):
                 }
     """
 
-    def __init__(self, backend: Backend):
+    def __init__(
+        self,
+        backend: Backend,
+        *,
+        acquire_alignment: Optional[int] = None,
```

Comment:
Hmm, do you think it is useful to have all these arguments? I was thinking just `round_pulse` would have a `enforce_minimum=False` option, so you could create an instance from the backend but get pulse durations not rounded up to the minimum if needed. I don't oppose the argument overrides, but I think the `enforce_minimum` (or a better name) option is still a good idea so that you don't need two `BackendTiming` instances to get both a real pulse duration and an incremental pulse duration.

Reply:
This could help us writing a test with some mock backend which doesn't provide constraints. I took an approach to modify constraint value to keep the method signature clean. Regarding ignoring the minimum condition, we need to modify not only `round_pulse` but also `pulse_time`. As we may introduce further controllability in future, e.g. ignore acquire alignment, then method would become much busier. As you mentioned, we need to define multiple instances for different controllability.

Reply:
Hmm, okay. I see your point. We can see how it goes. It might be worth more complexity in the BackendTiming class if it meant less complexity in the code using it.